cogs/moderation.py

```diff
@@ -131,6 +131,25 @@ def __init__(self, bot):
             await ctx.author.send(f"Could not send a warning to `{target}`.\n"
                                   "They may have DMs disables, or have blocked me.\n\n"
                                   f"Warning text was:\n{warning}")
+            
+    @command(hidden=True, aliases=['prune, p'])
+    async def purge(self, ctx, count: int):
+        """[p]purge <n> [@user] - purge <n> messages. Ping @user to only delete that user's messages of the last <n> messages. Multiple users also supported."""
+        if ctx.channel.permissions_for(ctx.author).manage_messages:
+            specific_users = [i.id for i in ctx.message.mentions]
```

Comment:
i dont like this. i think its a better idea to add an optional arg after count for a member converter so id and names work too and it isnt as hacky as this